src/server.c

```diff
@@ -183,6 +183,27 @@ server_global_filter(const struct wl_client *client, const struct wl_global *glo
 	return true;
 }
 
+/*
+ * This message is intended to help users who are trying labwc on
+ * clean/minimalist systems without existing Desktop Environments (possibly
+ * through Virtual Managers) where polkit is missing or GPU drivers do not
+ * exist, in the hope that it will reduce the time required to get labwc running
+ * and prevent some troubleshooting steps.
+ */
+static const char helpful_seat_error_message[] =
+"\n"
+"Some friendly trouble-shooting help\n"
+"===================================\n"
+"\n"
+"If a seat could not be created, this may be caused by lack of permission to the\n"
+"seat and video groups. If you are using a systemd setup, try installing polkit\n"
```

Comment:
`seat, input and video groups`. (Strictly speaking potentially also the `tty` group but mentioning just the 3 seems enough).